arm/assets/myoarm_nohand_assets.xml

```diff
@@ -0,0 +1,836 @@
+            <geom geom="TRI_cylinder" sidesite="TRI_cylinder_TRIlat_4_sidesite"/>
+            <site site="TRIlat_TRIlat-P5"/>
+        </spatial>
+        <spatial name="TRImed_tendon" class="upperarm_muscle">
+            <site site="TRImed_TRImed-P1"/>
+            <site site="TRImed_TRImed-P2"/>
+            <site site="TRImed_TRImed-P3"/>
+            <site site="TRImed_TRImed-P4"/>
+            <geom geom="TRI_cylinder" sidesite="TRI_cylinder_TRImed_4_sidesite"/>
+            <site site="TRImed_TRImed-P5"/>
+        </spatial>
+        <spatial name="ANC_tendon" class="upperarm_muscle">
+            <site site="ANC_ANC-P1"/>
+            <geom geom="ANC_ellipsoid_ANC" sidesite="ANC_ellipsoid_ANC_1_sidesite"/>
+            <site site="ANC_ANC-P2"/>
+        </spatial>
+        <spatial name="SUP_tendon" class="upperarm_muscle">
+            <site site="SUP_SUP-P1"/>
+            <site site="SUP_SUP-P2"/>
+            <geom geom="SUP_cylinder" sidesite="SUP_cylinder_SUP_2_sidesite"/>
+            <site site="SUP_SUP-P3"/>
+        </spatial>
+        <spatial name="BIClong_tendon" class="upperarm_muscle">
+            <site site="BIClong_BIClong-P1"/>
+            <site site="BIClong_BIClong-P2"/>
+            <geom geom="BIClong_ellipsoid_BIClong" sidesite="BIClong_ellipsoid_BIClong_2_sidesite"/>
+            <site site="BIClong_BIClong-P3"/>
+            <site site="BIClong_BIClong-P4"/>
+            <site site="BIClong_BIClong-P5"/>
+            <site site="BIClong_BIClong-P6"/>
+            <site site="BIClong_BIClong-P7"/>
+            <site site="BIClong_BIClong-P8"/>
+            <geom geom="Elbow_BIC_BRD_ellipsoid_BIClong" sidesite="Elbow_BIC_BRD_ellipsoid_BIClong_7_sidesite"/>
+            <site site="BIClong_BIClong-P10"/>
+            <site site="BIClong_BIClong-P11"/>
+        </spatial>
+        <spatial name="BICshort_tendon" class="upperarm_muscle">
+            <site site="BICshort_BICshort-P1"/>
+            <site site="BICshort_BICshort-P2"/>
+            <site site="BICshort_BICshort-P3"/>
+            <site site="BICshort_BICshort-P4"/>
+            <site site="BICshort_BICshort-P5"/>
+            <geom geom="Elbow_BIC_BRD_ellipsoid_BICshort" sidesite="Elbow_BIC_BRD_ellipsoid_BICshort_5_sidesite"/>
+            <site site="BICshort_BICshort-P6"/>
+            <site site="BICshort_BICshort-P7"/>
+        </spatial>
+        <spatial name="BRA_tendon" class="upperarm_muscle">
+            <site site="BRA_BRA-P1"/>
+            <geom geom="TRI_cylinder" sidesite="TRI_site_BRA_side"></geom>
+            <site site="BRA_BRA-P4"/>
+        </spatial>
+        <spatial name="BRD_tendon" class="upperarm_muscle">
+            <site site="BRD_BRD-P1"/>
+            <geom geom="ElbowBICBRD_ellipsoid_wrap" sidesite="Elbow_BIC_BRD_ellipsoid_BICshort_5_sidesite"/>
+            <site site="BRD_BRD-P2"/>
+            <site site="BRD_BRD-P3"/>
+        </spatial>
+
+
+
+       <!--  <spatial name="ECRL_tendon" class="forearm_muscle">
```

Comment:
let's remove the commented parrs

Reply:
@Vittorio-Caggiano I'll create a screenshot and a short description

Reply:
I've done this. See the latest file changes to [PR 232](https://github.com/MyoHub/myosuite/pull/232). I have added two descriptions/images, one for the model and one for the task.